docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx

```diff
@@ -484,24 +434,24 @@ import logo from '../public/logo.png'
 
 Alternatively, you can reference the public URL for the image asset based on the filename. For example, `public/logo.png` will serve the image at `/logo.png` for your application, which would be the `src` value.
 
-> **Warning:** If you're using TypeScript, you might encounter type errors when accessing the `src` property. You can safely ignore those for now. They will be fixed by the end of this guide.
+> **Warning:** If you're using TypeScript, you might encounter type errors when accessing the `src` property. To fix them, you need to add `next-env.d.ts` to the [`include` array](https://www.typescriptlang.org/tsconfig#include) of your `tsconfig.json` file. Next.js will automatically generate this file when you run your application on step 9.
```

Comment:
The `next-env.d.ts` file doesn't get auto-added to the `include` array by the TypeScript auto-configuration of `next dev`. There might be a bug with https://github.com/vercel/next.js/blob/79c934aaecae4acac6b5a635b9cf3bb95265a945/packages/next/src/lib/typescript/writeConfigurationDefaults.ts

Reply:
cc @samcx